indico/modules/categories/controllers/management.py

```diff
@@ -58,8 +60,22 @@ def _category_query_options(self):
         return children_strategy,
 
     def _process(self):
+        show_all_events = request.args.get('show_all') == '1'
+        page_number = int(request.args.get('page', 1))
+        order_columns = {'start_dt': Event.start_dt, 'title': db.func.lower(Event.title)}
+        direction = db.desc if request.args.get('desc', '1') == '1' else db.asc
+        order_column = order_columns[request.args.get('order', 'start_dt')]
+        events_paginated = (Event.query.with_parent(self.category)
+                            .order_by(direction(order_column)))
+        if not show_all_events:
+            events_paginated = events_paginated.paginate(page=page_number)
         return WPCategoryManagement.render_template('management/content.html', self.category, 'content',
-                                                    subcategories=self.category.children)
+                                                    subcategories=self.category.children,
+                                                    current_category=self.category, events_paginated=events_paginated,
+                                                    page_number=page_number,
+                                                    order_column=request.args.get('order', 'start_dt'),
+                                                    direction='desc' if request.args.get('desc', '1') == '1' else 'asc',
```

Comment:
very similar to `direction` earlier in the code. you could use this and then use this to get the direction for the query:

`.order_by(getattr(order_column, direction)())` (columns have desc/asc methods) or
`.order_by(getattr(db, direction)(order_column))`